telemetry/telemetrybuffer.go

```diff
@@ -60,7 +61,10 @@ const (
 	cni                = "CNI"
 )
 
-var payloadSize uint16 = 0
+var (
+	payloadSize                uint16 = 0
```

Comment:
I don;t think you need to set this to 0. GO inits variables ( like bool below ). You can choose to keep it / remove it.

Reply:
i didnt add this..i just moved this